pilot/docker/Dockerfile.proxy

```diff
@@ -14,5 +14,9 @@ COPY envoy_bootstrap_tmpl.json /var/lib/istio/envoy/envoy_bootstrap_tmpl.json
 # Allow non-root user to run envoy.
 RUN chmod 755 /usr/local/bin/envoy
 
+# We need to replacing http health-checking to a `curl localhost` command
+# when mTLS is enabled.
+RUN apt-get update && apt-get install -y curl
```

Comment:
change to
```
RUN apt-get update && \
    apt-get install --no-install-recommends -y curl && \
    rm -rf /var/lib/apt/lists/*
```
like https://github.com/istio/istio/blob/master/docker/Dockerfile.xenial_debug#L8-L23
(used to be in this _debug file)

ie don't leave a huge apt/lists behind and don't install stuff not strictly needed

Reply:
Done, thx!